geopandas/io/tests/test_file.py

```diff
@@ -461,8 +461,7 @@ def test_to_file_crs(tmpdir, engine, nybb_filename):
     assert result.crs == "epsg:3857"
 
     # specify CRS for gdf without one
-    df2 = df.copy()
-    df2.crs = None
+    df2 = df.copy().set_crs(None, allow_override=True)
```

Comment:
This copy is now redundant / duplicated as another one is done inside `set_crs` as inplace is false
```suggestion
    df2 = df.set_crs(None, allow_override=True)
```